components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -68,6 +68,7 @@ interface PanelHandler {
 
   ConnectToSite(array<string> accounts, string origin, int32 tab_id);
   CancelConnectToSite(string origin, int32 tab_id);
+  ClosePanelOnDeactivate(bool close);
```

Comment:
Please add a comment as to what this does. It's not totally clear from the function name.
edit: ok I understand that it sets a flag as to whether the panel should close when it loses focus. Is that correct? If so, perhaps we can call it something less like an action and more like a pref `setCloseOnDeactivate(bool shouldClose)`.

Reply:
done